reports/performance/_outliers.qmd

```diff
@@ -18,27 +18,29 @@ exclusively on these price-based factors that determine outlier status.
 ### Count by Type
 
 ```{r _outliers_type_breakdown}
-# Make new column for a few cells of the outliers section
+# Subset important outlier information
 training_data <- training_data %>%
-  mutate(price_outlier_reason = case_when(
-    # Also grabs outlier reason Non-livable computed in 00-ingest.R
-    str_detect(sv_outlier_reason1, regex("price|Non-livable", ignore_case = TRUE)) ~ sv_outlier_reason1,
-    str_detect(sv_outlier_reason2, regex("price", ignore_case = TRUE)) ~ sv_outlier_reason2,
-    str_detect(sv_outlier_reason3, regex("price", ignore_case = TRUE)) ~ sv_outlier_reason3,
-    TRUE ~ NA_character_
-  ))
+  mutate(
+    outlier_reasons_to_graph = if_else(
+      sv_is_outlier,
+      paste0(sv_outlier_reason1, " (", sv_outlier_reason2, ")"),
+      NA_character_
+    )
+  ) %>%
+  # Yank NA string values to make output cleaner
+  mutate(outlier_reasons_to_graph = str_remove_all(outlier_reasons_to_graph, " \\(NA\\)"))
```

Comment:
**suggestion (blocking):** Use a `case_when()` here. Creating a string and then manipulating it is fragile and error-prone.